Consul/Client_Options.cs

```diff
@@ -32,11 +32,18 @@ public class QueryOptions
         public static readonly QueryOptions Default = new QueryOptions()
         {
             Consistency = ConsistencyMode.Default,
+            Namespace = Environment.GetEnvironmentVariable("HTTPNamespaceEnvName"),
```

Comment:
This line stands out to me because the `Environment.GetEnvironmentVariable` is only used in one function in the whole of consuldotnet, so this seemed new. https://github.com/G-Research/consuldotnet/blob/4f9404755d73dc0d52f7edacf72c817143c30a91/Consul/Client.cs#L157

Is the namespace value always the same for all requests? If not, this environment variable approach might be an issue.

Reply:
Yeah, this is what the original Go Consul client does as well :/

Reply:
Oh this is my mistake. It seems they don't set any default Namespace.